econml/inference/_inference.py

```diff
@@ -828,9 +818,9 @@ class NormalInferenceResults(InferenceResults):
         The transform function to get the corresponding feature names from featurizer
     """
 
-    def __init__(self, d_t, d_y, pred, pred_stderr, inf_type, fname_transformer=lambda nm: nm,
+    def __init__(self, d_t, d_y, pred, pred_stderr, inf_type, fname_transformer=None,
                  feature_names=None, output_names=None, treatment_names=None):
-        self.pred_stderr = pred_stderr
+        self.pred_stderr = copy.deepcopy(pred_stderr)
```

Comment:
Same comment about copying

```suggestion
        self.pred_stderr = np.copy(pred_stderr)
```

Reply:
same as above